### ISSUE TITLE
GraphQL editor not saving properly

### ISSUE THREAD

**Issue by tipputhynhean** [getinsomnia/insomnia#2234]
If you type in the query manually, it get saved. But if you click on the suggested pop up, it doesn't get saved.

![Peek 2020-06-02 11-57](https://user-images.githubusercontent.com/20786858/83481804-ef188f00-a4c8-11ea-8b57-aff0e816530a.gif)

 - OS: Manjaro 20.0.2
 - Installation Method: AUR http://aur.archlinux.org/packages/insomnia
 - App Version

![image](https://user-images.githubusercontent.com/20786858/83482004-66e6b980-a4c9-11ea-9e76-854d8a339f9c.png)

**Comment by AndreyD** [comment]
+ have the same issue after update to Version 2020.2.0

**Comment by general-CbIC** [comment]
+1

Also, these fields disappear from the request.

**Comment by gschier** [comment]
Hmm, I cannot seem to reproduce this on my end. Can someone here do a bit more digging to see if there's a specific case where it happens and if it happens in other areas of the app? 🙏

**Comment by gschier** [comment]
Also... can you let me know if there are any errors in View > Toggle DevTools > Console?

**Comment by gschier** [comment]
Also, again... if anyone can share a public graphql endpoint that reproduces this, that would be great! 😁

**Comment by general-CbIC** [comment]
@gschier seems I got it.
If there is an error on schema validation, autocomplete works unpredictably.

I have this error in console when type any symbol in query

```
Uncaught Error: Names must match /^[_a-zA-Z][_a-zA-Z0-9]*$/ but "en-KE" does not.
```

In this case query not saves and when I try to send it, sends only part.
Switching to other query and back (like @tipputhynhean described in first message) leaves only part of query in editor.

**Comment by AndreyD** [comment]
> If there is an error on schema validation

+1
I had a types in schema without defined fields
After i fixed this, all works fine.

![Insomnia](https://user-images.githubusercontent.com/3584821/83760796-21b7b880-a67e-11ea-9518-d738d378e797.jpg)

**Comment by gschier** [comment]
Thanks for the details @general-CbIC and @AndreyD. It seems like updating our `graphql` dependency might do the trick but we should also add some extra safeguards around parse failures.

**Comment by anyforever** [comment]
I had the same problem
![screen capture image](https://transfer.sh/108YbX/Kapture%202020-06-07%20at%2019.50.30.gif)
please see here: https://transfer.sh/108YbX/Kapture%202020-06-07%20at%2019.50.30.gif

Insomnia:2020.2.1 (2020.2.1)
macOS:10.15.5 (19F101)
This problem was not found in the previous version of Insomnia

**Comment by gschier** [comment]
This should be fixed now in version 2020.2.2 https://insomnia.rest/changelog/core/2020.2.2
